code/modules/antagonists/ninja/weapons/energy_hankyu.dm

```diff
@@ -0,0 +1,219 @@
+/**
+ * # Energy hankyu
+ *
+ * The space ninja's hankyu.
+ *
+ * The hankyu that only space ninja spawns with. It's a bow with infinity arrows charging inside itself.
+ * Bow have 3 arrow types:
+ *		Damage arrow - give damage to target and knockdown it.
+ * 		EMP arrow - make emp around arrow hit.
+ * 		Repulse arrow - throws everyone away from turf where arrow hit.
+ * Also have scope.
+ */
+#define CHARGE_BASE 3 SECONDS
+/obj/item/gun/ballistic/bow/energy_hankyu
+	name = "energy hankyu"
+	desc = "Hi-Tech energy multi-bow"
```

Comment:
Again, a little lore with it. "Experienced Spider Clan assassins are torn on the fate of the hankyu. Some claim that it's arrows are too bulky and slow to compete with the standard-issue shuriken. however, hankyu users have repeatedly demonstrated the usefulness of the EMP and repulsive arrows against Nanotrasen mooks. Activate in hand to change the arrow payload.